README.md

```diff
@@ -360,30 +360,26 @@ Usage
 ------
 
 * Get java
-* Get maven 3
 
-* ``mvn compile`` will compile
-* ``mvn test`` will run the  unit tests
-* ``mvn package`` will package in a jar (found in target)
-* ``mvn install -DskipTests`` will check that you abide by the code style and that the code compiles. We enforce a strict style so that there is no debate as to the proper way to format the code.
+* ``./gradlew assemble`` will compile
+* ``./gradlew build`` will compile and run the  unit tests
+* ``./gradlew test `` will run the tests
+* ``./gradlew checkstyleTest`` will check that you abide by the code style and that the code compiles. We enforce a strict style so that there is no debate as to the proper way to format the code.
```

Comment:
This will only do the test code (src/test/java). `checkstyleMain` does `src/main/java` but maybe you want just `check` which is running the tests + all other things like checkstyle? Or if you want a one-shot task we could create one that depends on all tasks of type `Checkstyle`.

Reply:
Updated for "checkstyleMain"